src/features/lessons/add-lesson/add-lesson.js

```diff
@@ -102,36 +103,42 @@ export const AddLesson = () => {
     history.push(paths.LESSONS);
   }, [history]);
 
-  const onSubmit = (values) => {
-    const { lessonDate, themeName } = values;
-    const lessonVisits = formData.map((lessonVisit) => {
-      const {
-        presence, studentId, studentMark,
-      } = lessonVisit;
-      return (
-        {
-          comment: '',
-          presence,
-          studentId,
-          studentMark,
-        }
-      );
-    });
-
-    const mentorData = mentors.find((mentor) => mentor.email === mentorInput);
-
-    const theme = commonHelpers.capitalizeTheme(themeName);
-
-    const lessonObject = {
-      lessonDate,
-      themeName: theme,
-      lessonVisits,
-      studentGroupId: studentsGroup.id,
-      mentorId: mentorData.id,
-    };
-
-    if (!mentorsError && lessonObject) {
-      createLesson(lessonObject);
+  const onSubmit = async (values) => {
+    try {
```

Comment:
why we use try-catch element here?

Reply:
we make further validation of formData before submit 
123 await studentsFormDataValidation.validate(lessonVisits);
and catch possible errors. For extra safe

Reply:
Wanted to synchronize validation logic in add-lesson and edit-lesson, bz add-lesson has some problems with getting data into formData for submit from Formik. It goes beyond of this task, so I've got rid of changes in add-lesson and included field formData in editLessonValidation schema.